bleach/linkifier.py

```diff
@@ -171,11 +169,9 @@ def linkify(self, text):
         :raises TypeError: if ``text`` is not a text type
 
         """
-        if not isinstance(text, six.string_types):
+        if not isinstance(text, str):
             raise TypeError("argument must be of text type")
 
-        text = force_unicode(text)
```

Comment:
`force_unicode` checks text it coerces is valid utf-8, but the encoding arg for `str` uses `sys.getdefaultencoding` so maybe we should validate that.

Reply:
Oh, interesting! Nice catch!

Reply:
I don't think we need to worry about this. If the `text` value isn't a str, then we don't need to decode it so we don't need to validate encodings.

I think what you did is right and all we need to do is remove the `force_unicode` call.